packages/functions/package.json

```
@@ -0,0 +1,3 @@
+{
+  "name": "@vercel/functions"
```

This will need a changeset if you would like it to be published to npm. Also, since it's a new package, I suggest adding:

```suggestion
  "name": "@vercel/functions",
  "publishConfig": {
    "access": "public"
  }
```